x-pack/test/fleet_api_integration/apis/package_policy/create.ts

```diff
@@ -150,6 +165,27 @@ export default function (providerContext: FtrProviderContext) {
       expect(body.tags.find((tag: any) => tag.name === 'For File Tests').relationCount).to.be(9);
     });
 
+    it('should work with multiple policy ids', async function () {
+      const response = await supertest
+        .post(`/api/fleet/package_policies`)
+        .set('kbn-xsrf', 'xxxx')
+        .send({
+          name: 'filetest-6',
+          description: '',
+          namespace: 'default',
+          policy_ids: [agentPolicyId, agentPolicyId2],
+          enabled: true,
+          inputs: [],
+          package: {
+            name: 'filetest',
+            title: 'For File Tests',
+            version: '0.1.0',
+          },
+        })
+        .expect(200);
+      expect(response.body.item.policy_ids).to.eql([agentPolicyId, agentPolicyId2]);
```

Comment:
added new tests to cover create/update of package policy with multiple agent policy ids